packages/components/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 -   `isPrimary`, `isSecondary`, `isTertiary` and `isLink` props in `Button` have been deprecated. Use `variant` instead ([#31713](https://github.com/WordPress/gutenberg/pull/31713)).
 -   `isElevated` prop in `Card` has been deprecated. Use `elevation` instead ([#32566](https://github.com/WordPress/gutenberg/pull/32566)).
+-   `isScrollable` prop in `CardBody` default value changed from `true` to `false` ([#33490](https://github.com/WordPress/gutenberg/pull/33490))
```

Comment:
Noting that if we publish WordPress packages to npm from Guteberg 11.1 this week we will have to move up this entry and put into its own section.

Reply:
Is there also going to be a new release of the components package (e.g. 14.2.0)?

Reply:
Yes, we didn't have it for a while but now that WordPress 5.8 should be out tomorrow, there is no blocker.